pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,248 @@
+    "  name: GROUP_NAME\n",
+    "  annotations:\n",
+    "    kustomize.config.k8s.io/behavior: replace\n",
+    "users:\n",
+    "  - USER_1\n",
+    "  - USER_2\n",
+    "```\n",
+    "```{note}\n",
+    "The USER_n value is the email used to login via SSO in the preceeding steps.\n",
+    "```\n",
+    "Encrypt the file with sops. You can find the key to import from [here](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/prod/moc#secret-management):"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!sops --encrypt --encrypted-regex=\"^users$\" --pgp=\"0508677DD04952D06A943D5B4DC4116D360E3276\" groups/GROUP_NAME.yaml > groups/GROUP_NAME.enc.yaml\n",
+    "!grep \"fp: \" groups/GROUP_NAME.enc.yaml"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "If you or your team needs the ability to decrypt this file, you may include additional PGP key fingerprints in the sops command:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!sops --encrypt --encrypted-regex=\"^users$\" --pgp=\"0508677DD04952D06A943D5B4DC4116D360E3276, YOUR_GPG_KEY_FINGERPRINT\" groups/GROUP_NAME.yaml > groups/GROUP_NAME.enc.yaml\n",
+    "!grep \"fp: \" groups/GROUP_NAME.enc.yaml"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Explanation to the `sops` command:\n",
+    "- `encrypt` flag encrypts a resource\n",
+    "- `encrypted-regex` value maps to the XPath-like regex to specifies which parts of the file should be encrypted. The rest of the file is left as a plaintext for easier management. In this case we want to encrypt only the `users` property in the file. See the docs [here](https://github.com/mozilla/sops#encrypting-only-parts-of-a-file).\n",
+    "- `pgp` list all the GPG keys which will be used to encrypt this file.\n",
+    "\n",
+    "Don’t forget to remove the plaintext variant of the resource before staging for a commit:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!rm groups/GROUP_NAME.yaml"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "And list the sops encrypted file in the secret generator file:\n",
+    "```\n",
+    "# secret-generator.yaml\n",
+    "files:\n",
+    "---\n",
+    "- groups/GROUP_NAME.enc.yaml\n",
+    "```\n",
+    "Use sops to edit the encrypted group resource patch as:\n",
+    "```{note}\n",
+    "You can only do this if you included your pgp key in the step above, otherwise ensure all users are added before first encrypting the file.\n",
+    "```"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!sops groups/GROUP_NAME.enc.yaml"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
```

Comment:
Remove this, no sops is needed with Operate First SSO.

Reply:
Resolved in latest commit.